### ISSUE TITLE
insert_into() with add() does not support verbatim 

### ISSUE THREAD

**Issue by CJCombrink** [rbock/sqlpp11#489]
It seems like the following is valid:
```
const gen::PublicMyTable t;
db(insert_into(t).set(
   t.name        = "name",
   t.description = "description",
   t.vals        = sqlpp::verbatim<sqlpp::text>("ARRAY[2,3]")));
```

But not when using add:
```
auto multi_insert = insert_into(t).columns(t.name, t.description, t.vals);
multi_insert.values.add(
  t.name        = "name",
  t.description = "description",
  t.vals        = sqlpp::verbatim<sqlpp::text>("ARRAY[2,3]"));
db(multi_insert);
```

I get the following compiler error (gcc 10)
```
sqlpp11/0.61/_/_/package/5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9/include/sqlpp11/insert_value_list.h:408:42: error: no matching function for call to ‘sqlpp::insert_value_t<sqlpp::column_t<gen::PublicMyTable, gen::PublicMyTable_::Vals> >::insert_value_t(<brace-enclosed initializer list>)’
[build]   408 |         _data._insert_values.emplace_back(insert_value_t<lhs_t<Assignments>>{assignments._rhs}...);
```

My table is defined as 
```
CREATE TABLE MyTable
(
    name                TEXT,
    description         TEXT,
    vals                INTEGER[]
);
```
I have added the mapping of  `INTEGER[]` to `text` in ddl2cpp.

PS: There is another bug here where `MyTable` does not generate due to the array `INTEGER[]` (see #488)

**Comment by rbock** [comment]
I don't have a good solution for this, I am afraid. Multi line inserts values in a vector of tuples. For text fields they expect strings, but `verbatim_t` is not a string and not convertible to string (it would be serialized incorrectly if it were stored as string).

The quick workaround would be to use individual inserts.

A more long term solution could be 

- to introduce a way to change the tuple element for multi line insert, or
- to introduce array types

Both would be non-trivial and take a while, I think.

**Comment by ghais** [comment]
The same problem happens when doing insert_into with a select field

auto q = insert_into(table).columns(table.x);
q.values.add(table.x = select(foo.id).from(foo).where(foo.id = 3);

**Comment by rbock** [comment]
Yeah, that won´t work, because a sub select is not of type `int64_t` (or whatever the type of `x`is).

As mentioned above, the values are stored in a vector. But maybe that can be changed. I'll think about it.

**Comment by rbock** [comment]
Dynamic queries are different in https://github.com/rbock/sqlpp23. Using verbatim should work trivially.

But I won't change it in sqlpp11.
